.github/workflows/main.yml

```diff
@@ -23,21 +23,13 @@ jobs:
         extensions: ctype, iconv, intl, json, mbstring, pdo, pdo_sqlite
         coverage: none
 
-    - name: Checkout Lumen 6 Sample
```

Comment:
Lumen 6 is supported until September 6, 2022.

Reply:
Lumen 6 does not support PHP 8.